Documentation/guides/messages/xa0115.md

```diff
@@ -7,9 +7,13 @@ property will need to be updated to remove it before they will build
 successfully with newer versions of Xamarin.Android.  The newer armeabi-v7a ABI
 should now be used instead.  The `$(AndroidSupportedAbis)` setting can be found
 in the **Advanced** section of the **Android Options** project properties in
-Visual Studio and **Android Build** project properties in Visual Studio for Mac.
+Visual Studio and the **Advanced** tab of the **Android Build** project
```

Comment:
We have an *entire file* (and eventually URL) here. Let's take advantage of it. :-)

Please add separate `# Section`s so separately address Visual Studio for Windows vs. macOS.  Screenshots *may* be helpful.